Proxy/Configuration/Response/Json/index.ts

```diff
@@ -130,6 +130,21 @@ export namespace Json {
 			)
 		}, body)
 	}
+	export function reverseProcess(
+		configuration: Json,
+		variables: Record<string, any>,
+		body: Record<string, any>
+	): Record<string, any> {
+		return configuration.set.reduce((r, replacement) => {
+			if (typeof replacement == "string")
+				replacement = { find: replacement }
+			return Selector.set(
+				r,
+				replacement.find,
+				replacement.value == undefined ? undefined : replace(replacement.value, variables)
+			)
+		}, body)
```

Comment:
this is copied and pasted from the above function, the above one could just call this this